src/SFA.DAS.EmployerFinance.Web/Orchestrators/TransfersOrchestrator.cs

```diff
@@ -41,10 +41,11 @@ public async Task<OrchestratorResponse<IndexViewModel>> GetIndexViewModel(string
             var accountId = _hashingService.DecodeValue(hashedAccountId);
             var indexTask = _transfersService.GetCounts(accountId);
             var accountDetail = _accountApiClient.GetAccount(hashedAccountId);
+            var financialBreakdownTask = _transfersService.GetFinancialBreakdown(accountId);
```

Comment:
As appealing as it might be, you shouldn't make this request from here. The BFF pattern we're using means that each page in the UI should call one endpoint in the outer api (which in turn should only be referenced by one endpoint in the UI - a 1:1 ratio). This means you should add an endpoint to the outer api for this page if it doesn't already exist, and respond with all the information you need in its response.

Reply:
It looks as though an api endpoint for it might already exist:

```
_transfersService.GetCounts(accountId)
```

Ridiculous name though.

https://github.com/SkillsFundingAgency/das-apim-endpoints/blob/master/src/EmployerFinance/SFA.DAS.EmployerFinance.Api/Controllers/TransfersController.cs#L27